src/shapes/Group.ts

```diff
@@ -133,7 +131,13 @@ export class Group
 
     // perform initial layout
     this.layoutManager = layoutManager;
-    this.layoutManager.performLayout({
+    (
+      layoutManager ||
+      // legacy
+      new LayoutManager(
+        this.clipPath ? new ClipPathLayout() : new FixedLayout()
+      )
```

Comment:
i don't understand this change here

Reply:
aparently some tests were failing because I made FitContentLayout not respect width/height of constructor options
Add to that that `layoutManager` is now optional (which is a good decision IMO) then use this fallback for the initial layout

Reply:
But this part of the initial fixed layout is still not stable